README.md

```diff
@@ -1,16 +1,24 @@
-# agora_rtc_engine
+# Agora RTC Engine for Flutter
```

Comment:
I think `Agora RTC SDK for Flutter` should be better

Reply:
Agreed! Updating now